src/Raygun.NetCore.Blazor/Models/RequestDetails.cs

```diff
@@ -36,7 +36,7 @@ internal record RequestDetails
         /// The HTTP method used to request the URL (GET, POST, PUT, etc).
         /// </summary>
         [JsonInclude]
-        public HttpMethod HttpMethod { get; set; }
```

Comment:
This was a bit strange, not sure if it was a typo or on purpose. The JSON parser failed trying to convert this. The rest of the code did not complain after the change.